swap/src/protocol/alice/event_loop.rs

```diff
@@ -144,6 +148,11 @@ where
                 swarm_event = self.swarm.next_event() => {
                     match swarm_event {
                         SwarmEvent::Behaviour(OutEvent::SpotPriceRequested { request, channel, peer }) => {
+                            if self.resume_only {
+                                tracing::warn!(%peer, "Ignoring spot price request from {} because ASB started in resume-only mode", peer);
+                                continue;
```

Comment:
We should return a proper error the CLI here instead of just dropped the channel.

Reply:
Yeah, we should handle all `expected` errors properly and return a proper message, see https://github.com/comit-network/xmr-btc-swap/pull/461/commits/84626d654f8c904a5588a9fdcd2782f4d0a8d4bc